routes/index.js

```diff
@@ -3,18 +3,32 @@ const router = express.Router()
 const passport = require('passport')
 const User = require('../models/user')
 const Article = require('../models/article')
-const {isLoggedIn} = require('../middleware')
+const {isLoggedIn, checkCaptcha} = require('../middleware')
+const validator = require('validator')
+const svgCaptcha = require('svg-captcha')
+const csrf = require('csurf')
+const rateLimiter = require('express-rate-limit')
+
+const csrfProtection = csrf({ cookie: true });
+
+const authLimit = rateLimiter({
+    windowMs: 60 * 60 * 1000,
```

Comment:
Is this really an hour? Looks like 60h

Reply:
1000ms times 60 is one minute, times 60 again for an hour